src/ripple/app/ledger/impl/LedgerConsensus.cpp

```diff
@@ -401,10 +401,10 @@ class LedgerConsensusImp
 
         // Adjust tracking for each peer that takes this position
         std::vector<NodeID> peers;
-        for (auto& it : mPeerPositions)
+        for (auto& i : mPeerPositions)
```

Comment:
Nice change, since `i` is no longer an iterator.  But `i` doesn't seem quite right to me either.  Perhaps `peerPos`?